modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc

```diff
@@ -605,8 +605,7 @@ For an example, see <<import-tool-multiple-input-files-regex-example>>.
 |
 
 |--normalize-types[=true\|false]
-|Whether or not to normalize property types to Cypher types, e.g. `int` becomes `long` and `float` becomes `double`.
-|true
+|When true, non-array property values will be converted to their equivalent Cypher types. For example, all integer values will be converted to 64-bit long integers. This takes up more space on disk, but avoids Cypher converting the type during queries.
```

Comment:
```suggestion
|When `true`, non-array property values will be converted to their equivalent Cypher types. For example, all integer values will be converted to 64-bit long integers. This takes up more space on disk but avoids Cypher converting the type during queries.
```

Reply:
The descriptions must be updated in the code base as well. Otherwise, the test will fail.